setup.py

```diff
@@ -59,7 +59,7 @@ def get_environment_marker_support_level():
 def main():
     extras_require = {}
     install_requires = [
-        "py>=1.5.0",
+        "py>=1.6.0",
```

Comment:
We learned the hard way before to not change dependency requirements in bug-fix releases. 😬 

Any chance of staying compatible with `py-1.5`, but using the new functionality from `py-1.6` when that's installed?

Reply:
If not possible/desired, we should target `features` instead.